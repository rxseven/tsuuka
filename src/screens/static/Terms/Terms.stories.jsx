import React from 'react';
import { storiesOf } from '@storybook/react';

import Terms from './index';

const title = 'Screens/Static/Terms';
const component = <Terms />;

storiesOf(title, module).add('default', () => component);
