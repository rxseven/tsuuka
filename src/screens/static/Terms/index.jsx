import React from 'react';

import { Body, Document, Head } from 'components/common/base/Page';
import Card from 'components/common/base/Card';
import Layout from 'components/common/composite/Layout';

function Terms() {
  return (
    <Document>
      <Head>
        <title>Terms and Conditions of Use · 通貨換算ツール</title>
        <meta content="Tsūka Terms and Conditions of Use" name="description" />
      </Head>
      <Body>
        <Layout adaptive>
          <Card>
            <Card.Body>
              <Card.Title>Terms and Conditions of Use</Card.Title>
              <Card.Text>
                Lorem Ipsum is simply dummy text of the printing and typesetting
                industry. Lorem Ipsum has been the industry’s standard dummy
                text ever since the 1500s, when an unknown printer took a galley
                of type and scrambled it to make a type specimen book. It has
                survived not only five centuries, but also the leap into
                electronic typesetting, remaining essentially unchanged. It was
                popularised in the 1960s with the release of Letraset sheets
                containing Lorem Ipsum passages, and more recently with desktop
                publishing software like Aldus PageMaker including versions of
                Lorem Ipsum.
              </Card.Text>
            </Card.Body>
          </Card>
        </Layout>
      </Body>
    </Document>
  );
}

export default Terms;
