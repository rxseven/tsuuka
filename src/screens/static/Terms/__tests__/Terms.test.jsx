import { shallow } from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';

import Terms from '../index';

describe('<Terms />', () => {
  // Arrange
  const component = <Terms />;

  describe('Unit tests', () => {
    it('should render without crashing', () => {
      // Act
      const wrapper = shallow(component);

      // Assert
      expect(wrapper).toBeDefined();
    });
  });

  describe('Snapshot tests', () => {
    it('should render correctly', () => {
      // Act
      const tree = renderer.create(component).toJSON();

      // Assert
      expect(tree).toMatchSnapshot();
    });
  });
});
