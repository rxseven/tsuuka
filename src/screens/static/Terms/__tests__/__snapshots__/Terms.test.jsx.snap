// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Terms /> Snapshot tests should render correctly 1`] = `
<div
  className="row justify-content-sm-center"
>
  <div
    className="col-md-10 col-lg-8"
  >
    <div
      className="card text-text-left mb-3"
    >
      <div
        className="card-body"
      >
        <h5
          className="card-title"
        >
          Terms and Conditions of Use
        </h5>
        <p
          className="card-text"
        >
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industryâ€™s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
        </p>
      </div>
    </div>
  </div>
</div>
`;
